app-misc/francis/francis-9999.ebuild

```
@@ -0,0 +1,29 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KDE_ORG_CATEGORY="utilities"
+KFMIN=6.5.0
+QTMIN=6.6.0
+inherit ecm gear.kde.org
+
+DESCRIPTION="Francis uses the well-known Pomodoro technique to help you get more productive."
```

```suggestion
DESCRIPTION="Productivity application using the well-known pomodoro technique"
```

`${PN}` shall never be part of `DESCRIPTION`, and we don't write full sentences there either, so no full stop either.